### ISSUE TITLE
is it possible to generate a report to a device without binding (TZ-487)

### ISSUE THREAD

**Issue by diazmanuel** [espressif/esp-zigbee-sdk#179]
### Question

Hello, I am trying to generate a report so that it reaches one device periodically and the other only for changes.

To achieve this, it occurred to me to configure the report after declaring the cluster in the following way. 
```
    esp_zb_device_register(esp_zb_ep_list);

    esp_zb_zcl_attr_location_info_t filter;
    esp_zb_zcl_reporting_info_t *report;
    filter.attr_id=CUSTOM_OT_CLUSTER_ATTR_ERROR_ID;
    filter.cluster_id=CUSTOM_OT_CLUSTER_ID;
    filter.endpoint_id=ZIGBEE_BOILER_ENDPOINT;
    filter.cluster_role=ESP_ZB_ZCL_CLUSTER_SERVER_ROLE;
    filter.manuf_code=ESP_ZB_ZCL_ATTR_NON_MANUFACTURER_SPECIFIC;
    report=esp_zb_zcl_find_reporting_info(filter);
    if(report!=NULL){
        report->u.send_info.def_max_interval=REPORTABLE_MAX_INTERVAL_OT_S;
        report->u.send_info.def_min_interval=REPORTABLE_MIN_INTERVAL_OT_S;
        report->u.send_info.max_interval=REPORTABLE_MAX_INTERVAL_OT_S;
        report->u.send_info.min_interval=REPORTABLE_MIN_INTERVAL_OT_S;
        report->u.send_info.delta.u32=REPORTABLE_DELTA_OT;
        if(esp_zb_zcl_update_reporting_info(report)!=ESP_OK) ESP_LOGE(TAG, "REPORTE NO CONFIGURADO");
    }else ESP_LOGE(TAG, "CONFIGURACION DE REPORTE NO ENCONTRADA");
```

In this way, if I perform a binding of that cluster, the device will send periodically and by changes to the first device. For the second device I would have to find another way to do it since if I did the bind in the same way it would receive the reports periodically (which is what I want to avoid)

reading the documentation I found a function that generates a report, try to generate said report in the following way without performing a binding
```
        esp_zb_zcl_report_attr_cmd_t cmd_req;
        cmd_req.cluster_role=ESP_ZB_ZCL_CLUSTER_SERVER_ROLE;
        cmd_req.attributeID=CUSTOM_OT_CLUSTER_ATTR_ERROR_ID;
        cmd_req.clusterID=CUSTOM_OT_CLUSTER_ID;
        cmd_req.address_mode=ESP_ZB_ZDO_BIND_DST_ADDR_MODE_16_BIT_GROUP;
        cmd_req.zcl_basic_cmd.dst_endpoint=ZIGBEE_HUB_ENDPOINT;
        cmd_req.zcl_basic_cmd.src_endpoint=ZIGBEE_BOILER_ENDPOINT;
        cmd_req.zcl_basic_cmd.dst_addr_u.addr_short=ZIGBEE_HUB_SHORT_ADDRESS;

        esp_zb_zcl_report_attr_cmd_req(&cmd_req);
```
but it did not work since the second device did not receive said report

In short, I am trying to generate a report under different conditions for 2 different devices, could you tell me what I am doing wrong, why the reporting force does not work and is there a way to achieve what I am looking for?
Is the report configuration only one per cluster-attribute or can I have more than one depending on a destination mac address?

**Comment by xieqinan** [comment]
Hello @diazmanuel ,

The operation of binding is necessary for attribute reporting.

The ESP-Zigbee-SDK provides the `esp_zb_zcl_config_report_cmd_req()` function, which can facilitate report configuration from the remote device. Does this meet your requirements?
